### PR TITLE
[PR] Adicionado ngModel e ngChange

### PR DESCRIPTION
**Pull request by walkerdepaula**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
       inputEnter: true
     });
   }
-  
+
   function demoController() {
     var self = this;
-  
+
+    self.test = {};
```
Comment (sougiovn) [inline src/main.js]: Remover

File: src/ng-currency-range-slider.js [hunk 2]
```diff
@@ -64,94 +64,101 @@
         max: '=',
         valueFormatter: '=',
         valueParser: '=',
-        range: '=',
+        ngModel: '=',
         onRangeChange: '&'
```
Comment (sougiovn) [inline src/ng-currency-range-slider.js]: Trocar por `ngChange`

File: src/ng-currency-range-slider.js [hunk 3]
```diff
@@ -64,94 +64,101 @@
...
       }
-      
+
       function emitRangeChange(event) {
         range = event.detail;
-        if (isNotUndefined($scope.range)) {
-          $scope.range.min = range.min;
-          $scope.range.max = range.max;
+
+        $scope.onRangeChange({
```
Comment (sougiovn) [inline src/ng-currency-range-slider.js]: Trocar por `ngChange`

**Review by sougiovn — CHANGES_REQUESTED** [review]

File: src/main.js (hunk 1 above)
Comment (walkerdepaula) [reply src/main.js]: Fix in b266fbd

File: src/ng-currency-range-slider.js (hunk 2 above)
Comment (walkerdepaula) [reply src/ng-currency-range-slider.js]: Fix in b266fbd

File: src/ng-currency-range-slider.js (hunk 3 above)
Comment (walkerdepaula) [reply src/ng-currency-range-slider.js]: Fix in b266fbd

**Review by sougiovn — APPROVED** [review]

**Pull request merged**